src/test/scala/io/iohk/ethereum/ledger/LedgerTestSetup.scala

```diff
@@ -393,15 +393,15 @@ trait MockBlockchain extends MockFactory { self: TestSetupWithVmAndValidators =>
       .once()
   }
 
-  def setHeaderByHash(hash: ByteString, header: Option[BlockHeader]): CallHandler1[ByteString, Option[BlockHeader]] =
-    (blockchain.getBlockHeaderByHash _).expects(hash).returning(header)
+  def setHeaderInChain(hash: ByteString, result: Boolean = true): CallHandler1[ByteString, Boolean] = {
```

Comment:
brackets are not needed, and this method is not used anywhere

Reply:
it is used in BranchResolutionSpec

Reply:
ah right, I was looking at the line 404 removal. do you know why it's no longer called there?

Reply:
now it's being called within the specific tests, so the steps taken to solve it are visible

Reply:
clear, only brackets left